src/server_manager/web_app/ui_components/app-root.js

```diff
@@ -229,12 +229,49 @@ export class AppRoot extends mixinBehaviors
       #appDrawer a:focus {
         outline: none;
       }
+      #links-footer {
+        margin-top: 36px;
+      }
+      #language-icon {
+        padding-top: 10px;
+      }
+      #language-dropdown {
+        padding-left: 22px;
```

Comment:
This is not RTL-friendly. Does the left padding need to be different from the right padding?

Reply:
Let me look into it a bit. I didn't do anything special because it looked to be already handled.
<img width="1363" alt="Screen Shot 2020-06-16 at 1 09 07 PM" src="https://user-images.githubusercontent.com/1009390/84805857-e2b92980-afd2-11ea-8166-f7d7635ed8b4.png">